config/schedule.rb

```diff
@@ -127,6 +127,9 @@ def local(time)
   if Flipper.enabled?(:monthly_supplies_reports)
     runner "QuestionnaireResponses::MonthlySuppliesReports.new.seed"
   end
+  if Flipper.enabled?(:drug_stock_questionnaires)
```

Comment:
We are not using `:drug_stock` flipper flag, because that is already enabled in all the deployments. Instead, we've introduced a new flag to mark drug stocks' migration to dynamic forms.